src/project/ProjectNameEditable.tsx

```diff
@@ -83,7 +58,7 @@ const ProjectNameEditable = ({
     <Text
       key="text"
       cursor={clickToEdit ? "pointer" : undefined}
-      onClick={clickToEdit ? handleEdit : undefined}
+      onClick={() => (clickToEdit ? actions.editProjectName() : undefined)}
```

Comment:
Can this be `{clickToEdit ? actions.editProjectName : undefined}`

?

Reply:
Same as comment above, [`6b7b162`](https://github.com/microbit-foundation/python-editor-next/pull/763/commits/6b7b162c9c7399d766a75310e87e258351a08f55).